lib/common/ipc.c

```diff
@@ -579,20 +579,18 @@ crm_ipc_prepare(uint32_t request, xmlNode * message, struct iovec ** result, uin
 
             free(buffer);
 
-            if (header->size_compressed > biggest) {
-                biggest = 2 * QB_MAX(header->size_compressed, biggest);
-            }
+            biggest = QB_MAX(header->size_compressed, biggest);
```

Comment:
Or `biggest = QB_MAX(4 * header->size_compressed, biggest);` which would not warn if the following message is 1 byte larger